uber/custom_tags.py

```diff
@@ -9,6 +9,18 @@ def datetime(dt, fmt='%-I:%M%p %Z on %A, %b %e'):
 
 
 @register.filter
+def shift_end(dt, duration):
+    curdate = dt + timedelta(hours=int(duration))
+    fmt = "%Y-%m-%dT%H:%M:%S"
+    return ' '.join(curdate.astimezone(c.EVENT_TIMEZONE).strftime(fmt).split()).replace('AM', 'am').replace('PM', 'pm')
+
+
+@register.filter
+def list_length(tgtlist):
+    return len(tgtlist)
```

Comment:
There's already a ``|length`` filter which does the same thing as this.  I don't think this is necessary, unless I'm missing something.